pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -133,14 +142,16 @@ def prepare() {
       sh """/bin/bash
       echo "multi_node_aio_prepare.prepare/Prepare RPC Configs"
       set -xe
-      scp -r -o StrictHostKeyChecking=no /opt/rpc-openstack deploy1:/opt/
-      scp -o StrictHostKeyChecking=no ${env.WORKSPACE}/user_zzz_gating_variables.yml deploy1:/etc/openstack_deploy/user_zzz_gating_variables.yml
+      scp -r -o StrictHostKeyChecking=no /opt/rpc-openstack infra1:/opt/
+      scp -o StrictHostKeyChecking=no ${env.WORKSPACE}/user_zzz_gating_variables.yml infra1:/etc/openstack_deploy/user_zzz_gating_variables.yml
 
-      ssh -T -o StrictHostKeyChecking=no deploy1 << 'EOF'
+      ssh -T -o StrictHostKeyChecking=no infra1 << 'EOF'
       set -xe
       sudo cp /etc/openstack_deploy/user_variables.yml /etc/openstack_deploy/user_variables.yml.bak
       sudo cp -R /opt/rpc-openstack/openstack-ansible/etc/openstack_deploy /etc
       sudo cp /etc/openstack_deploy/user_variables.yml.bak /etc/openstack_deploy/user_variables.yml
+      # Write random var to user_variables.yml incase it's empty (which happens to be the case on stable/mitaka)
+      echo "osa_ops_mnaio: true" | sudo tee -a /etc/openstack_deploy/user_variables.yml
```

Comment:
We could upstream this, however I'm not sure it's worthwhile since stable/mitaka isn't really used upstream any more.  Thoughts?

Reply:
seems like a good idea to get that in when the deploy osa playbook is executed.

Reply:
Thanks for upstreaming @cloudnull, will adapt once that merges!